app/views/users/show.html.erb

```diff
@@ -1,9 +1,18 @@
 <div class="container">
     <%= render partial: "user", locals: { user: @user } %>
+
+  <div class="gap-1" style="display:flex; justify-content: start; width:100%;">
     <div class="user-card gap-0 flex-col items-start border-card">
         <span class="card-title">Bio</span>
         <span class="user-bio"><%= @user.bio %></span>
     </div>
+    <a
+      href="<%= user_new_posts_path(user_id: @user.id) %>" 
+      class="border-card user-card justify-center card-title" 
+      style="background-color:black; color:white;">
+      Add New Post
+    </a>
```

Comment:
- [ ] Hello @abdozayan12 I just noticed the first suggestion I made is not being displayed I'm not sure If this is a bug from GitHub but I wasn't able to create a post due to the undefined path specified at line `10` as `user_new_posts_path` which is creating the below error when clicking on a user card 
  ![firefox_EJqDxXc1sC](https://github.com/abdozayan12/DataApp/assets/104968236/8c3d0182-b83d-48a2-b5e9-edf7042cf038)
- [ ] Since this is a crucial requirement for your app to work correctly I'd suggest working on it my apologies for the inconvenience.